### PR TITLE
Visual docs updates for overlay mode

### PR DESCRIPTION
**Pull request by marstupa**
### Description
Updates for overlay mode in Visual testing

### Motivation and Context
To keep up with feature updates that impact the UI

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/visual-testing/workflows/review.md [hunk 1]
```diff
@@ -137,21 +137,23 @@ The user interface including the actions you can take are described in [User Int
...
+Selecting one of the builds allows you to get to the "Diff Review" page, where you can Accept or Reject detected diffs.
 
-<img src={useBaseUrl('/img/sauce-visual/DiffReviewPage.png')} alt="Diff Review-overlay"/>
+<img src={useBaseUrl('/img/sauce-visual/diff-review-page.jpg')} alt="Diff review page with toolbars"/>
 
-By default, you will see diffs in a side-by-side view, which lets you easily compare the current diff against the Baseline. On the right side, you will see a toolbar enabling you to review the diffs. You can:
+By default, you will see diffs in a Side-by-side view, which lets you easily compare the Current diff against the Baseline. On the right side, you will see a toolbar enabling you to review the diffs. You can:
 
-- Switch from side-by side-to focused view for easier comparison between the Baseline and Current snapshot.
+- Switch from Side-by side-to Overlay view for easier comparison between the Baseline and Current snapshot.
```
Comment (FriggaHel) [inline docs/visual-testing/workflows/review.md:146]: Shouldn't this be 
```suggestion
- Switch from Side-by-side to Overlay view for easier comparison between the Baseline and Current snapshot.
```

File: docs/visual-testing/workflows/review.md [hunk 2]
```diff
@@ -137,21 +137,23 @@ The user interface including the actions you can take are described in [User Int
...
 - Zoom in or out of the snapshots.
 - Zoom to 100% or fit snapshot to the screen.
 - Highlight or hide diffs for easier discovery of the changes.
 
 There are conditional actions you can use, depending on the type of view you have activated:
 
-- If in side-by-side view, you can mirror the zoom and pan actions from the Current snapshot to the Baseline.
-- If in focused view, you can toggle between Baseline and Current diff.
+- If in Side by side view, you can mirror the zoom and pan actions from the Current snapshot to the Baseline.
```
Comment (FriggaHel) [inline docs/visual-testing/workflows/review.md:153]: Same here for consistency
```suggestion
- If in Side-by-side view, you can mirror the zoom and pan actions from the Current snapshot to the Baseline.
```

**Review by FriggaHel — APPROVED** [review]

File: docs/visual-testing/workflows/review.md [hunk 3]
```diff
@@ -137,21 +137,23 @@ The user interface including the actions you can take are described in [User Int
...
+Selecting one of the builds allows you to get to the "Diff Review" page, where you can Accept or Reject detected diffs.
 
-<img src={useBaseUrl('/img/sauce-visual/DiffReviewPage.png')} alt="Diff Review-overlay"/>
+<img src={useBaseUrl('/img/sauce-visual/diff-review-page.jpg')} alt="Diff review page with toolbars"/>
 
-By default, you will see diffs in a side-by-side view, which lets you easily compare the current diff against the Baseline. On the right side, you will see a toolbar enabling you to review the diffs. You can:
+By default, you will see diffs in a Side-by-side view, which lets you easily compare the Current diff against the Baseline. On the right side, you will see a toolbar enabling you to review the diffs. You can:
 
-- Switch from side-by side-to focused view for easier comparison between the Baseline and Current snapshot.
+- Switch from Side-by-side to Overlay view for easier comparison between the Baseline and Current snapshot.
```
Comment (paweltomaszewskisaucelabs) [inline docs/visual-testing/workflows/review.md:146]: It's now on the top of diff stage

Comment (paweltomaszewskisaucelabs) [reply docs/visual-testing/workflows/review.md:146]: So maybe it should be removed from context of "On the right side"

Comment (marstupa) [reply docs/visual-testing/workflows/review.md]: Good catch, thanks! I have updated it.

File: docs/visual-testing/workflows/review.md [hunk 4]
```diff
@@ -137,21 +137,22 @@ The user interface including the actions you can take are described in [User Int
...
-- Switch from side-by side-to focused view for easier comparison between the Baseline and Current snapshot.
 - Zoom in or out of the snapshots.
 - Zoom to 100% or fit snapshot to the screen.
 - Highlight or hide diffs for easier discovery of the changes.
 
 There are conditional actions you can use, depending on the type of view you have activated:
 
-- If in side-by-side view, you can mirror the zoom and pan actions from the Current snapshot to the Baseline.
-- If in focused view, you can toggle between Baseline and Current diff.
+- If in Side-by-side view, you can mirror the zoom and pan actions from the Current snapshot to the Baseline.
```
[the remark this answers is not in the document]
Comment (marstupa) [reply docs/visual-testing/workflows/review.md:152]: You mean as a UX improvement or here in docs?

[the remark this answers is not in the document]
Comment (marstupa) [reply docs/visual-testing/workflows/review.md:152]: Made a note, thanks!

**Pull request merged**